distributed/diagnostics/plugin.py

```diff
@@ -186,6 +189,15 @@ def release_dep(self, dep, state, report):
         """
 
 
+def _get_worker_plugin_name(plugin) -> str:
+    """Returns the worker plugin name. If plugin has no name
+    it assigns a random name using funcname and uuid"""
```

Comment:
Nitpick: Explicitly mentioning `funcname` and `uuid` in this docstring might be a little too specific as these are really implementation details. Maybe just mention that a random name is generated? 

```suggestion
    """ Returns the worker plugin name. If plugin has no name attribute a random name is used. """
```